R/signal.R

```diff
@@ -22,14 +22,27 @@
 #'
 #' @keywords internal
 #' @export
-signal_experimental <- function(when, what) {
-  signal(
-    "",
-    "lifecycle_experimental",
-    when = when,
-    what = what
-  )
+signal_experimental <- function(what) {
```

Comment:
I removed `when` because I don't think it helps either the developer or the user. It's marked experimental, and doesn't appear to be [used anywhere](https://github.com/search?l=R&q=signal_experimental&type=Code), so I think this is safe to change.

If you feel strongly about it for consistency with the `deprecate_` functions, I don't mind keeping it.